src/ui/handlers/abstract-ui-handler.ts

```diff
@@ -18,25 +28,140 @@ export abstract class UiHandler {
     this.mode = mode;
   }
 
-  abstract setup(): void;
+  /**
+   * Whether the handler is currently active. Can be made inactive by calling {@linkcode stop}.
+   */
+  public get active(): boolean {
+    return this._active;
+  }
 
-  show(..._args: unknown[]): boolean {
-    this.active = true;
+  /**
+   * Whether the handler is setup and ready to be displayed through calling {@linkcode start}.
+   * note: unused for now since all handlers are ready at all times.
+   */
+  public get ready(): boolean {
+    return this._ready;
+  }
```

Comment:
I think @DayKev said that the docs for get/set should be on the field definition as they are "inherited" by the getter/setter